integrations/visual_studio/conan_extension.rst

```diff
@@ -0,0 +1,37 @@
+|visual_logo| Conan Extension for Visual Studio
```

Comment:
Same, this is for sure not ready for 2.0, so better drop it, or just put a placeholder

Reply:
I think it's useful information even if it can't be used at the moment. I added a warning to inform it's not ready yet

Reply:
How about now? It's just a warning saying that it's not working with Conan 2.x, and a link to the marketplace in case you're still using Conan 1.x